tools/generate-gltf-classes/generate.js

```diff
@@ -71,6 +78,8 @@ function generate(options, schema) {
 
         ${headers.map((header) => `#include ${header}`).join("\n")}
 
+        using namespace CesiumUtility;
```

Comment:
This is not required for all files (e.g. classes that inherit from `NamedObject` instead of `ExtensibleObject`) but I don't see the harm in keeping it around.

Reply:
And eventually `NamedObject` may make its way into `CesiumUtility`